app/services/flow/base_flow.rb

```diff
@@ -31,6 +31,15 @@ def handle(step)
       wrap_send(handler)
     end
 
+    def mark_step_complete(step)
+      return if step.nil?
```

Comment:
cc @stevegsa as he may know more

This method is implemented on the [base step](https://github.com/18F/identity-idp/blob/master/app/services/flow/base_step.rb#L19-L22), so my initial thought is it would be preferable to implement it on the `DocAuthBaseStep` or the relevant classes that inherit from there, e.g. `MobileFrontImageStep`, etc. if different behavior is needed?

Reply:
I added this because I wanted a way to mark steps complete given the flow and just the name of the step, not the step itself.